### PR TITLE
HRTI: use system clock for S

### PR DESCRIPTION
**Pull request by tjol**
This would make my test for HRTI pass :-)

### CODE REVIEW

**Comment by cwesson** [comment]
Interesting... HRTI passes for me using steady_clock.  I wonder if stead_clock is implemented differently on different systems.

File: src/include/funge_types.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <cstdint>
+#include <cstddef>
```
Comment (cwesson) [inline src/include/funge_types.h:10]: What is cstddef needed for here?

**Comment by tjol** [comment]
I bet it is. The C++ standard says it has no relation to wall time, and on Linux it appears it doesn’t, but that doesn’t mean it *can’t* be related to wall time.

**Review by cwesson — APPROVED** [review]

File: src/include/funge_types.h (hunk 1 above)
Comment (tjol) [reply src/include/funge_types.h:10]: I had to include it for `size_t`

**Pull request merged**